gstreamer: broken due to bison?

It seems the recent update of bison to 3.0.4 broke gstreamer:
```
make[4]: Entering directory '/builddir/gstreamer-0.10.36/gst/parse'
  GEN    lex.priv_gst_parse_yy.c
  CC     libgstparse_la-lex.priv_gst_parse_yy.lo
lex.priv_gst_parse_yy.c:1009:5: warning: redundant redeclaration of 'priv_gst_parse_yyget_column' [-Wredundant-decls]
 
     ^
lex.priv_gst_parse_yy.c:4:19: note: previous declaration of 'priv_gst_parse_yyget_column' was here
 static inline int priv_gst_parse_yyget_column  (void * yyscanner);
                   ^
lex.priv_gst_parse_yy.c:1011:6: warning: redundant redeclaration of 'priv_gst_parse_yyset_column' [-Wredundant-decls]
 
      ^
lex.priv_gst_parse_yy.c:5:20: note: previous declaration of 'priv_gst_parse_yyset_column' was here
 static inline void priv_gst_parse_yyset_column (int  column_no , void * yyscanner);
                    ^
  CC     libgstparse_la-grammar.tab.lo
grammar.tab.c: In function 'priv_gst_parse_yyparse':
grammar.tab.c:67:25: error: too few arguments to function 'priv_gst_parse_yylex'
 #define yylex           priv_gst_parse_yylex
                         ^
grammar.tab.c:1821:16: note: in expansion of macro 'yylex'
   if (yychar == YYEMPTY)
                ^
./grammar.y:39:5: note: declared here
 int priv_gst_parse_yylex (void * yylval_param , yyscan_t yyscanner);
     ^
```
I can't make too much sense of the messages...

--- a/srcpkgs/gstreamer/patches/fix_bison_grammar_y.patch
+++ b/srcpkgs/gstreamer/patches/fix_bison_grammar_y.patch
@@ -1,0 +1,11 @@
+--- gst/parse/grammar.y	2011-12-30 02:14:35.000000000 +0100
++++ gst/parse/grammar.y	2015-08-04 17:40:31.981404808 +0200
+@@ -36,7 +36,7 @@
+ 
+ typedef void* yyscan_t;
+ 
+-int priv_gst_parse_yylex (void * yylval_param , yyscan_t yyscanner);
++int priv_gst_parse_yylex (void * yylval_param);
+ int priv_gst_parse_yylex_init (yyscan_t scanner);
+ int priv_gst_parse_yylex_destroy (yyscan_t scanner);
+ struct yy_buffer_state * priv_gst_parse_yy_scan_string (char* , yyscan_t);

--- a/srcpkgs/gstreamer/template
+++ b/srcpkgs/gstreamer/template
@@ -1,7 +1,7 @@
 # Template file for 'gstreamer'.
 pkgname=gstreamer
 version=0.10.36
-revision=5
+revision=6
 build_style=gnu-configure
 configure_args="--disable-valgrind --enable-docbook"
 hostmakedepends="libtool pkg-config flex python perl docbook-xsl glib-devel"

